src/open_inwoner/components/templates/components/AnchorMenu/AnchorMenu.html

```diff
@@ -8,6 +8,16 @@
             {% for anchor in anchors %}
                 <li class="anchor-menu__list-item">
                     <a class="link" href="{{ anchor.0 }}">{{ anchor.1 }}</a>
+                    <!-- subheadings -->
+                    {% if anchor.2 %}
+                        <ul class="anchor-menu__sublist anchor-menu__sublist--mobile">
+                            {% for subheading in anchor.2 %}
+                                <li>
```

Comment:
In order to make this work more specifically with the JS Gumshoe library you need to give each HTML tag its own class, preferably with BEM; so this LI could have an 'anchor-menu__sublist--item class' or something; so you can target them separately.